### PR TITLE
OTel Metrics API Update

### PR DESCRIPTION
**Pull request by lukebarrett24**

### CODE REVIEW

File: data-ingestion-apis/opentelemetry-metrics.mdx [hunk 1]
```diff
@@ -42,56 +42,119 @@ curl -X POST "https://demo.middleware.io/v1/metrics" \
...
-                }
-            ]
+                  }
+                ]
+              }
+            }
+          ]
         }
-    ]
+      ]
```
Comment (eduardo-mw) [inline data-ingestion-apis/opentelemetry-metrics.mdx:153]: Should instead be 
```json 
"scope_metrics": [
    {
        "metrics": [
            {
                "name": "uploads.count",
                "unit": "1",
                "description": "Count of user uploads",
                "sum": {
                    "aggregationTemporality": 2,
                    "isMonotonic": true,
                    "dataPoints": [
                        {
                            "asInt": 5,
                            "startTimeUnixNano": 1711559739191393000,
                            "timeUnixNano": 1711559739191393000,
                            "attributes": [
                                {
                                    "key": "uploads.count.user_email",
                                    "value": {
                                        "stringValue": "jane@domain.com"
                                    }
                                }
                            ]
                        }
                    ]
                }
            },
            {
                "name": "connections.active",
                "unit": "1",
                "description": "Number of active connections",
                "gauge": {
                    "dataPoints": [
                        {
                            "asInt": 215,
                            "timeUnixNano": 1711559739191393000,
                            "attributes": [
                                {
                                    "key": "connections.active.hostname",
                                    "value": {
                                        "stringValue": "vm-prod-2"
                                    }
                                }
                            ]
                        }
                    ]
                }
            },
            {
                "name": "downloads.speeds",
                "unit": "By",
                "description": "Download speeds of users in a particular region",
                "histogram": {
                    "aggregationTemporality": 2,
                    "dataPoints": [
                        {
                            "startTimeUnixNano": 1711559739191393000,
                            "timeUnixNano": 1711559739191393000,
                            "count": 6,
                            "sum": 258,
                            "bucketCounts": [
                                1,
                                1,
                                1,
                                1,
                                0,
                                0,
                                0,
                                0,
                                1,
                                1
                            ],
                            "explicitBounds": [
                                0,
                                10,
                                20,
                                30,
                                40,
                                50,
                                60,
                                70,
                                80,
                                90,
                                100
                            ],
                            "min": 0,
                            "max": 98,
                            "attributes": [
                                {
                                    "key": "region",
                                    "value": {
                                        "stringValue": "us-west"
                                    }
                                }
                            ]
                        }
                    ]
                }
            }
        ]
    }
]
```

**Review by eduardo-mw — CHANGES_REQUESTED** [review]

File: data-ingestion-apis/opentelemetry-metrics.mdx [hunk 2]
```diff
@@ -42,56 +42,130 @@ curl -X POST "https://demo.middleware.io/v1/metrics" \
...
-                        "value": {
-                            "string_value": "xxxxxxxxxx"
-                        }
-                    },
-                    {
-                        "key": "mw.resource_type",
-                        "value": {
-                            "string_value": "custom"
-                        }
+  "resourceMetrics": [
```
Comment (Zamichel) [inline data-ingestion-apis/opentelemetry-metrics.mdx:45]: Did this change recently? It was resource_metrics

**Comment by Zamichel** [comment]
Is it possible to make the json code blocks collapsable in some way? Maybe use accordians w/ some heading that allows a user to open it. The new example although good is very long and it makes the page pretty massive.

File: data-ingestion-apis/opentelemetry-metrics.mdx (hunk 2 above)
Comment (lukebarrett24) [reply data-ingestion-apis/opentelemetry-metrics.mdx:45]: Not sure. @eduardo-mw can you confirm?

**Comment by lukebarrett24** [comment]
No way to create collapsable in Mintlify but we can do it in Markdoc 😎

File: data-ingestion-apis/opentelemetry-metrics.mdx (hunk 2 above)
Comment (lukebarrett24) [reply data-ingestion-apis/opentelemetry-metrics.mdx:45]: Adding @bhogayatakb to this conversation to double check

**Review by Zamichel — APPROVED** [review]

File: data-ingestion-apis/opentelemetry-metrics.mdx (hunk 2 above)
Comment (bhogayatakb) [reply data-ingestion-apis/opentelemetry-metrics.mdx:45]: This should be `resource_metrics`, See Reference [here](https://github.com/open-telemetry/opentelemetry-proto/blob/main/opentelemetry/proto/metrics/v1/metrics.proto#L44)

Comment (eduardo-mw) [reply data-ingestion-apis/opentelemetry-metrics.mdx:45]: @lukebarrett24  Once that change is made I can approve asap

**Pull request merged**